### PR TITLE
still trying to get windows CI working

### PR DESCRIPTION
**Pull request by hdgarrood**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-language: c
-dist: trusty # because of perf issues
-sudo: required
+language: node_js
+node_js:
+  - "10"
```
Comment (JordanMartinez) [inline .travis.yml]: Did you want 10 or just the latest version?

```
language: node_js
  - "node"
  # ^ use latest version of NodeJS
```

Comment (hdgarrood) [reply .travis.yml]: I want "10" because I want to opt in to upgrades later on, I don't want them to happen automatically.

Comment (JordanMartinez) [reply .travis.yml]: Makes sense.

**Comment by hdgarrood** [comment]
This is finally working, and I can delete appveyor. :tada:

**Pull request merged**